packages/expo-av/src/AV.types.ts

```diff
@@ -196,6 +196,13 @@ export type AVPlaybackStatusSuccess = {
    * with `didJustFinish` set to `true`. `didJustFinish` is never `true` in any other case.
    */
   didJustFinish: boolean;
+  /**
+   * An array of subtitles that are included in the HLS stream.
+   * > Note: This is only available on Android when the video is loaded using ExoPlayer.
+   * @platform android
+   * @platform ios
+   */
+  subtitles?: string[];
```

Comment:
So Android Media Player doesn't support HLS streams afaik, and handles subtitles very differently to how ExoPlayer and AVPlayer (on iOS) do it. 

On top of that, web support for subtitles is going to be quite different as well. 

Don't know what your thoughts are on this @Kudo?

Reply:
I prefer leaving that type as optional. I understand that the user must check if subtitles are present every time, even if they know they will be there. However, if someone writes code without reading documentation and deduces behavior based on the type system, he won't know if `subtitles` will always exist. The empty array does not provide as much context as an optional field in the object because the array may be empty. After all, we don't have subtitles at the current moment.

Reply:
> On top of that, web support for subtitles is going to be quite different as well.

> The empty array does not provide as much context as an optional field in the object because the array may be empty. 

both these points make sense to me. thanks for having the better idea.